temporal/api/cloud/namespace/v1/message.proto

```diff
@@ -0,0 +1,89 @@
+syntax = "proto3";
+
+package temporal.api.cloud.namespace.v1;
+
+option go_package = "go.temporal.io/api/cloud/namespace/v1;namespace";
+
+import "google/protobuf/timestamp.proto";
+
+message CertificateFilterSpec {
+    // The common_name in the certificate
+    string common_name = 1;
+    // The organization in the certificate
+    string organization = 2;
+    // The organizational_unit in the certificate
+    string organizational_unit = 3;
+    // The subject_alternative_name in the certificate
+    string subject_alternative_name = 4;
+}
+
+message CodecServerPropertySpec {
+    // Server endpoints
+    string endpoint = 1;
+    // Whether to pass access token, i.e. jwt
+    bool pass_access_token = 2;
+    // Whether to include credentials
+    bool include_credentials = 3;
+}
+
+message NamespaceSpec {
+    // The region where the namespace is (or will be) active
+    string region = 1;
+    // The base64 encoded ca cert(s) that the clients can use for authentication and authorization
+    string accepted_client_ca = 2;
+    // The number of days the workflows data will be retained for
+    int32 retention_days = 3;
```

Comment:
align to the UI - retention_period could add days in the field name as well if desired

Reply:
ack fixed.